src/utils/data_loaders.py

```diff
@@ -173,7 +174,13 @@ def process_sentence(tokenizer_name, sent, max_seq_len, sos_tok=SOS_TOK, eos_tok
     '''process a sentence '''
     max_seq_len -= 2
     assert max_seq_len > 0, "Max sequence length should be at least 2!"
-    TOKENIZER = AVAILABLE_TOKENIZERS[tokenizer_name]
+    if "bert" in tokenizer_name:
+        import pdb; pdb.set_trace()
+        TOKENIZER = BertTokenizer(tokenizer_name)
```

Comment:
does this re-create the BERT tokenizer at every `process_sentence` call? Does that make preprocessing really slow?

Reply:
Yeah, this is going to be painful. See comment above about `lru_cache`.